tests/test_ida_features.py

```diff
@@ -37,7 +37,7 @@ def check_input_file(wanted):
 def get_ida_extractor(_path):
     check_input_file("5f66b82558ca92e54e77f216ef4c066c")
 
-    # have to import import this inline so pytest doesn't bail outside of IDA
+    # have to import this inline so pytest doesn't bail outside of IDA
```

Comment:
This feels like an errant doubling. But it's possible it isn't. (This heuristic is something I manually consider/apply)